Themis/Models/Exercise.swift

```diff
@@ -22,8 +22,8 @@ struct Exercise: Codable {
     let assessmentType: String?
     let problemStatement: String?
     let gradingInstructions: String? // For Programming Assesments this is nil
-    let templateParticipation: Participation
-    let solutionParticipation: Participation
+    // let templateParticipation: Participation check again
```

Comment:
Why is this commented out?

Reply:
As mentioned above. Decoding didn't work properly as both were not available in the response. So commented out as a quick fix. @tomrudnick and @tcw0 will fix this properly in the future